lib/elixir/lib/module/types/of.ex

```diff
@@ -128,7 +128,7 @@ defmodule Module.Types.Of do
     context = remote(struct, :__struct__, 0, meta, context)
 
     entries =
-      for key <- Map.keys(struct.__struct__()), key != :__struct__ do
+      for key <- Map.keys(struct.__struct__()) |> :lists.sort(), key != :__struct__ do
```

Comment:
this one is only going to affect error messages, right? Maybe we should order in the error messages then?

Reply:
The error message seems to already be sorted here: https://github.com/elixir-lang/elixir/blob/main/lib/elixir/lib/module/types.ex#L257.
I will just fix tests to remove the influence of ordering there: https://github.com/elixir-lang/elixir/pull/12418/commits/16bba32940397016b6d7d911b4a2d4206dd765ae